apis/fluent/atrium-api-fluent/src/commonTest/kotlin/ch/tutteli/atrium/api/fluent/en_GB/CharSequenceToContainAtLeastElementsOfExpectationsTest.kt

```diff
@@ -0,0 +1,149 @@
+package ch.tutteli.atrium.api.fluent.en_GB
+
+import ch.tutteli.atrium.creating.Expect
+import ch.tutteli.atrium.api.verbs.internal.testFactory
+import ch.tutteli.atrium.logic.creating.typeutils.IterableLike
+import ch.tutteli.atrium.specs.emptyIterableLikeTypes
+import ch.tutteli.atrium.specs.integration.AbstractCharSequenceToContainAtLeastExpectationsTest
+import ch.tutteli.atrium.testfactories.TestFactory
+import kotlin.collections.component1
+import kotlin.collections.component2
+
+class CharSequenceToContainAtLeastElementsOfExpectationsTest : AbstractCharSequenceToContainAtLeastExpectationsTest(
+    getAtLeastElementsOfTriple(),
+    getAtLeastIgnoringCaseElementsOfTriple(),
+    getAtLeastButAtMostElementsOfTriple(),
+    getAtLeastButAtMostIgnoringCaseElementsOfTriple(),
+    getNotToContainPair(),
+    getExactlyPair(),
+    Companion::getErrorMsgAtLeastButAtMost
+) {
+
+    companion object : CharSequenceToContainSpecBase() {
+
+        @TestFactory
+        fun toContainAtLeastOneElementsOf() =
```

Comment:
you need to move them from the companion into the regular CharSequenceToContainAtLeastElementsOfExpectationsTest (i.e. above companion object). Otherwise they will not be executed as part of the Test (I guess). Try to make them fail (e.g. by changing `atLeast(1)` to `atLeast(2)` you will see that they are not executed).